packages/lustre-v6/lustre-v6.6.101.7/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "The Lustre V6 Verimag compiler"
+description: """
+This package contains:
+  - lv6: the (current) name of the compiler (and interpreter via -exec)
+  - the lustre-v6 ocaml lib: allows to call the Lustre v6 interpreter from ocaml
+  - the lustre-v6 rdbg plugin: allows to debug Lustre v6 program wth rdbg.
+"""
+maintainer: "erwan.jahier@univ-grenoble-alpes.fr"
+authors: "Erwan Jahier and Pascal Raymond"
+license: "CeCILL"
+homepage: "http://www-verimag.imag.fr/lustre-v6.html"
+bug-reports: "http://www-verimag.imag.fr/lustre-v6.html"
+depends: [
+  "ocaml" {>= "4.05"}
+  "base-unix"
+  "extlib" {build} | "extlib-compat" {build}
+  "lutils" {>= "1.49"}
+  "oasis" {build & >= "0.4.7"} | "oasis-mirage" {build & >= "0.4.7"}
+  "ocamlbuild" {build}
+  "ocamlfind"
+  "rdbg" {>= "1.184"}
+]
+build: [
+  [make "build"]
+  ["oasis" "setup"] {with-test}
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"] {with-test}
+  ["ocaml" "setup.ml" "-build"] {with-test}
+  ["ocaml" "setup.ml" "-test"] {with-test}
+]
+install: [make "install"]
+remove: [
```

Comment:
This is no longer needed with `opam2`. It tracks the installed files and knows what to do with uninstall (similarly in the other opam files)

Reply:
ok, cool. I will remove it.